docs/en/contributing-to-appium/appium-from-source.md

```diff
@@ -80,36 +75,37 @@ have to modify your `/etc/authorization` file in one of two ways:
 2. Run the following command which automatically modifies your
    `/etc/authorization` file for you:
 
-    ```center
+    ```
     npm install -g authorize-ios
     sudo authorize-ios
 	```
 
 At this point, run:
 
-```center
-rm -rf node-modules
-npm install
-gulp transpile
+```
+rm -rf node-modules && npm install
```

Comment:
should we include package-lock.json in the rm?

Reply:
Good point. I'll add a little blurb. I think _most_ repos now don't produce them, but I may have missed some.

Reply:
I also appear to have used `node-modules` instead of `node_modules`. So doubly good catch!